fiftyone/core/odm/dataset.py

```diff
@@ -457,6 +457,7 @@ class DatasetDocument(Document):
     skeletons = DictField(EmbeddedDocumentField(KeypointSkeleton))
     default_skeleton = EmbeddedDocumentField(KeypointSkeleton)
     sample_fields = EmbeddedDocumentListField(SampleFieldDocument)
+    dataset_schema = EmbeddedDocumentListField(SampleFieldDocument)
```

Comment:
I'm not sure this is necessary. Especially if this feature is only with respect to the current view's schema

Reply:
agreed, cleaning it up